### PR TITLE
need to bundle with the fixed version 5.2 of objectweb.arm because th…

### PR DESCRIPTION
**Pull request by jtydhr88**
…e it doesn't exist in Oxygen 3a but sapphire need it

### CODE REVIEW

**Comment by jtydhr88** [comment]
jenkins retest

**Comment by jtydhr88** [comment]
jenkins retest

**Comment by yanan** [comment]
jenkins retest

File: tools/features/com.liferay.ide.hidden/feature.xml [hunk 1]
```diff
@@ -158,7 +158,7 @@
          id="org.objectweb.asm"
          download-size="0"
          install-size="0"
-         version="0.0.0"
+         version="5.2.0.v20170126-0011"
```
Comment (gamerson) [inline tools/features/com.liferay.ide.hidden/feature.xml]: I don't think we need to specify this one.  We should keep the versions removed.

**Comment by gamerson** [comment]
if we need at least a specific version we can specify it in another way.

**Pull request closed without merging**